### PR TITLE
Reorganized Menu Shortcut

### PR DESCRIPTION
**Pull request by zo-shin**
1 - For languages in Latin letters, made shortcuts closest to English.
2 - For other languages, added shortcuts in brackets at the end.

### CODE REVIEW

File: lang/pt_BR.ts [hunk 1]
```diff
@@ -273,7 +273,6 @@
     </message>
     <message>
         <source>D&amp;epends on size</source>
-        <translatorcomment>Não entendi a diferença entre essa string e a opção &quot;Auto&quot;. Traduzir para &quot;Depende(r) do tamanho&quot; irá passar a mesma impressão do modo automático; afinal, ele também depende do tamanho disponível.</translatorcomment>
```
Comment (zaps166) [inline lang/pt_BR.ts:276]: Why removing comments?

**Review by zaps166 — COMMENTED** [review]
`ru` and `uk` shouldn't be changed, too (according to other apps):
Screenshot_20220505_135404

**Comment by zo-shin** [comment]
As you see in the picture, there are no ФПД on the keyboard. If such letters are used as shortcut, they definitely don’t work.
As for the comment, those languages have been ignored for a long time. If there is someone who wants to revise them, those comments won’t do much work. If you resist the clean-up, I will get them back.

**Comment by zaps166** [comment]
> As you see in the picture, there are no ФПД on the keyboard. If such letters are used as shortcut, they definitely don’t work. As for the comment, those languages have been ignored for a long time. If there is someone who wants to revise them, those comments won’t do much work. If you resist the clean-up, I will get them back.

It's not true, if you change keyboard layout to e.g. Ukrainian, you can use these letters and key shortcuts are working.

File: lang/pt_BR.ts (hunk 1 above)
Comment (zaps166) [reply lang/pt_BR.ts:276]: @zo-shin ping here

**Comment by zaps166** [comment]
Merged from command line

**Pull request closed without merging**